### DISCUSSION TITLE
Substitute author name for blog posts with website title

### DISCUSSION THREAD

**Discussion by valerius21** [retorquere/zotero-better-bibtex#2218] [Q&A] [answered]
Hi  there,

Thank you for providing this great project!

When I want to include websites/blog posts in my research and save them with Zotero, the author's name is often missing. I was thinking of replacing the empty author's field with the website title if no name is recognized. Do you have any idea how to accomplish that?

Thanks!

![image](https://user-images.githubusercontent.com/10323945/180244736-04ec331b-7652-4137-ad93-6847e68835de.png)

**Comment by retorquere** [comment, marked as answer]
Can be done in a [postscript](https://retorque.re/zotero-better-bibtex/exporting/scripting/).
